proto/smartknob.proto

```diff
@@ -230,8 +251,18 @@ message MotorCalibration {
     uint32 pole_pairs = 4;
 }
 
+message StrainState {
+    int32 press_weight = 1;
+    float press_value = 2;
```

Comment:
I'm not clear why this is a float and it's initialized to 2.

Reply:
whats after the equals sign is not actual values but its "position" i think in the protobuf message

Its a float since its been changed from earlier implementation, its a value between 0.00 and 1.00 where 1 represents press weight acheived (250g)